doc/CMakeLists.txt

```diff
@@ -57,6 +57,8 @@ if (DOXYGEN_FOUND)
 
 	find_package(Perl)
 
+	set (MARKDOWN_LINK_CONVERTER ${CMAKE_BINARY_DIR}/bin/markdownlinkconverter)
```

Comment:
See cmake/Modules/LibAddMacros.cmake (e.g. exporterrors), you should use find_util and DEPENDS. This also works in the CMAKE_CROSSCOMPILING case.